### PR TITLE
Typo corrections and some readability improvements

### PR DESCRIPTION
**Pull request by Bratdaking**
Hey all,

Saw some minor typo's in two pages already pulled. So hereby a new request with the corrections, only minor changes. Feel free to add your own additions as well.

Cheers,
Bart

PS. Something I noticed in a lot of our pages. Markdown ignores line endings within paragraphs. In the end it will render it into one paragraph. If you have a paragraph that consists of several lines that each should go on a new line then you either need to make it into an ordered list (start with a - or *, or a number), or add double spaces at the end (double spaces and a line-end act as a hard line-end).

### CODE REVIEW

File: docs/fieldnorm/README.md [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+However, the ISO-15189 does not include bioinformatic or software development processes. 
+Fortunately, the IEC-62304 exists, which is a harmonized standard about "Medical device software – Software life cycle processes". 
+The use of this standard remains voluntary, however, it gives good guidance to demonstrate compliance with legal requirements. 
 
-We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state of the art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16). 
+We took the IEC-62304 as a guidance to create a fieldnorm for bioinformatics, to make sure that the software we create is developed and manufactured in accordance with the state-of-the-art taking into account the principles of development life cycle, risk management, including verification and validation, as required by the IVDR (Annex I, point 16).
```
Comment (rhpvorderman) [inline docs/fieldnorm/README.md:10]: This correction is incorrect. State of the art is not hyphenated. See also https://en.wikipedia.org/wiki/State_of_the_art

File: docs/fieldnorm/README.md [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+While writing this fieldnorm, it became clear that the size of bioinformatics teams differs. 
+In particular, when there is only a single bioinformatician working for the medical laboratory, some of the items described in this fieldnorm will be rather impossible. For example, reviewing each others code will be difficult. 
+While the fieldnorm does describe the needs to comply to the IVDR and the ISO-15189, it might be impossible to comply to the fieldnorm due to the lack of personnel. 
+Make sure choices about no compliance are reasoned and documented, and revise these choices once the team has expanded. 
 
-This field norm does not include Personnel (6.2 in ISO-15189:2022), as this should be covered by QMS of the Medical Laboratory already. 
+This fieldnorm does not include Personnel (6.2 in ISO-15189:2022), as this should be covered by QMS of the Medical Laboratory already.
```
Comment (rhpvorderman) [inline docs/fieldnorm/README.md:20]: So I did some research on this correction and it turns out that in English, this `field norm` with a space. It also turns out that this is a concept in physics: https://en.wikipedia.org/wiki/Field_norm

A better word would be "field standard" as I see in the suggestions to translate "veldnorm". 

Something to discuss on Friday, I guess.

File: docs/fieldnorm/README.md (hunk 1 above)
Comment (Bratdaking) [reply docs/fieldnorm/README.md:10]: You are completely right, although the reason not entirely 😅:
It should be hyphenated when it is used as an adjective, and I thought incorrectly that it was used together with principles so state-of-the-art principles, but it used as a Noun, so you are right it should be written without.

File: docs/fieldnorm/README.md (hunk 2 above)
Comment (Bratdaking) [reply docs/fieldnorm/README.md:20]: I stumbled upon the same problem seeing the space. But in the rest of the document and the referrals (like our nice abstract) we always spoke about fieldnorn. Therefor I just removed the space to keep things at least the same. But I completely agree, field standard would be way better.

File: docs/fieldnorm/README.md (hunk 1 above)
Comment (rhpvorderman) [reply docs/fieldnorm/README.md:10]: Ah thanks, I learned something new.

File: docs/fieldnorm/README.md (hunk 2 above)
Comment (rhpvorderman) [reply docs/fieldnorm/README.md:20]: I agree that consistency is better until we agree to use a different term.

File: docs/fieldnorm/README.md (hunk 1 above)
Comment (hwmvandeutekom) [reply docs/fieldnorm/README.md:10]: I think I copied this from the IVDR

**Review by rernst — APPROVED** [review]
Approved, let's review everything in final review.

**Pull request merged**